nebula/ui/components/MZHelpSheet.qml

```diff
@@ -0,0 +1,240 @@
+/*
+  MZHelpSheet {
+      title: "MZHelpSheet"
+      iconSource: "qrc:/ui/resources/connection-info-dark.svg"
+
+      model: [
+          {type: MZHelpSheet.BlockType.Title, text: "title"},
+          {type: MZHelpSheet.BlockType.Text, margin: 8, text: "text"},
+          {type: MZHelpSheet.BlockType.Text, margin: 16, text: "text"},
+          {type: MZHelpSheet.BlockType.Text, margin: 16, text: "text"},
+          {type: MZHelpSheet.BlockType.PrimaryButton, text: "Primary", margin: 16, action: () => { sheet.close(); MZNavigator.requestScreen(VPN.ScreenGetHelp) } },
+          {type: MZHelpSheet.BlockType.LinkButton, text: "Link", margin: 8, action: () => { MZUrlOpener.openUrl("https://mozilla.org") } }
+      ]
+    }
+*/
+
+MZBottomSheet {
+    id: bottomSheet
+
+    property alias iconSource: icon.source
+    property alias title: title.text
+    property var model
+
+
+    enum BlockType {
+        Title,
+        Text,
+        PrimaryButton,
+        LinkButton
+    }
+
+    implicitHeight: Math.min(contentItem.implicitHeight, maxHeight)
+
+    contentItem: ColumnLayout {
+        spacing: 0
+
+        ColumnLayout {
+            id: headerLayout
+
+            Layout.topMargin: 8
+
+            spacing: 0
+
+            RowLayout {
+                spacing: 0
+
+                Item {
+                    Layout.topMargin: MZTheme.theme.windowMargin / 2
+                    Layout.leftMargin: MZTheme.theme.windowMargin
+                    Layout.preferredHeight: 24
+                    Layout.preferredWidth: 24
+                    Layout.alignment: Qt.AlignTop
+
+                    Image {
+                        id: icon
+                        anchors.centerIn: parent
+
+                        sourceSize.width: MZTheme.theme.iconSize
+
+                        mirror: MZLocalizer.isRightToLeft
+                        fillMode: Image.PreserveAspectFit
+                    }
+                }
+
+
+                MZBoldLabel {
+                    id: title
+
+                    Layout.topMargin: MZTheme.theme.windowMargin / 2
+                    Layout.leftMargin: 8
+                    Layout.alignment: Qt.AlignTop
+
+                    verticalAlignment: Text.AlignVCenter
+                    lineHeightMode: Text.FixedHeight
+                    lineHeight: 24
+                }
+
+                Item {
+                    Layout.fillWidth: true
+                }
+
+                MZIconButton {
+                    id: iconButton
+
+                    Layout.rightMargin: MZTheme.theme.windowMargin / 2
+
+                    Layout.preferredHeight: MZTheme.theme.rowHeight
+                    Layout.preferredWidth: MZTheme.theme.rowHeight
+
+                    //                skipEnsureVisible: true // prevents scrolling of lists when this is focused
+
+                    onClicked: bottomSheet.close()
+
+                    accessibleName: MZI18n.GlobalClose
+                    Accessible.ignored: !visible
```

Comment:
This line should not be needed because MZButtonBase has Accessible.ignored: !visible